.github/workflows/pylint.yml

```diff
@@ -20,5 +20,11 @@ jobs:
         run: python3 -m pip install --requirement utils/cq/requirements.txt
       - name: Install enchant
         run: sudo apt-get update && sudo apt-get -y install enchant
+      - name: Pull via git
+        run: git fetch origin ${{ github.event.pull_request.base.sha }}
+      - name: See git diff
+        run: git diff ${{ github.event.pull_request.base.sha }} -U10
```

Comment:
This is debugging only so could come out but it's also not doing any harm for now.